dttools/src/jx.h

```diff
@@ -216,4 +216,63 @@ int jx_is_constant( struct jx *j );
 /** Export a jx object as a set of environment variables.  @param j A JX_OBJECT. */
 void jx_export( struct jx *j );
 
+/** Iterate over the values in an array.
+ * The iteation state is stored by the caller in an opaque pointer variable. When starting iteration,
```

Comment:
Small spelling mistake, second word. Happens in the other two descriptions as well.